docs/sites.yml

```
@@ -3507,3 +3507,10 @@
   categories:
     - Marketing
     - IOT
+- title: freeCodeCamp curriculum
+  main_url: "https://learn.freecodecamp.org"
+  url: "https://learn.freecodecamp.org"
+  featured: false
+  description: Learn to code with free online courses, programming projects, and interview preparation for developer jobs.
+  categories:
+    - Web development
```

```suggestion
    - Web Development
    - Learning
```

(Casing difference + Learning category)